Translations/hr.po

```diff
@@ -7,18 +7,18 @@ msgid ""
 msgstr ""
 "Project-Id-Version: DevilutionX\n"
 "POT-Creation-Date: 2021-09-20 04:34+0200\n"
-"PO-Revision-Date: 2021-09-20 04:34+0200\n"
-"Last-Translator: gogo <linux.hr@protonmail.com>\n"
+"PO-Revision-Date: 2021-09-24 15:55+0200\n"
+"Last-Translator: gogo <trebelnik2@gmail.com>\n"
 "Language-Team: \n"
 "Language: hr\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 3.0\n"
+"X-Generator: Poedit 2.3\n"
 "Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
 "%10<=4 && (n%100<12 || n%100>14) ? 1 : 2);\n"
 "X-Poedit-SourceCharset: UTF-8\n"
-"X-Poedit-KeywordsList: _;N_;P_:1c,2\n"
+"X-Poedit-KeywordsList: _;N_\n"
 "X-Poedit-Basepath: ..\n"
```

Comment:
This change will cause Poedit to not pickup monster and spell names when updating the translation from source